File: src/test/java/com/github/weewar/mapviewer/service/impl/key/TerrainImageKeyTest.java
package com.github.weewar.mapviewer.service.impl.key;

import com.github.weewar.mapviewer.service.impl.keys.TerrainImageKey;
import nl.jqno.equalsverifier.EqualsVerifier;
import org.junit.Test;

public class TerrainImageKeyTest {
    @Test
    public void verifyEqualsContract() {
        EqualsVerifier.forClass(TerrainImageKey.class).verify();
    }
}

File: src/test/kotlin/com/github/weewar/mapviewer/service/impl/keys/TerrainImageKeyTest.kt
package com.github.weewar.mapviewer.service.impl.keys

import nl.jqno.equalsverifier.EqualsVerifier
import org.junit.jupiter.api.Test

class TerrainImageKeyTest {
    @Test
    fun `verify equals contract`() {
        EqualsVerifier.forClass(TerrainImageKey::class.java).verify()
    }
}